lib/browser/objects-registry.js

```diff
@@ -55,13 +57,14 @@ class ObjectsRegistry {
   }
 
   // Clear all references to objects refrenced by the WebContents.
-  clear (contextId) {
-    let owner = this.owners[contextId]
+  clear (webContents, contextId) {
+    const webContentsContextId = `${webContents.id}-${contextId}`
+    let owner = this.owners[webContentsContextId]
     if (!owner) return
 
     for (let id of owner) this.dereference(id)
 
-    delete this.owners[contextId]
+    delete this.owners[webContentsContextId]
```

Comment:
Since we already have this, I think we can avoid the extra property lookup by saying `delete owner` instead?

Reply:
To delete a key in dictionary this is the only supported syntax.